### PR TITLE
Add playlist feature

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
Added Playlist feature

Files to check:
- playlistCard.jsx
- VideosInPlaylist.jsx
- Playlist.jsx

functions:
- postPlaylist.js
- postPlaylist.js
- getPlaylistFromBackend.js

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
Code looks clean ,good to merge

File: src/components/cards/PlaylistCard.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const image = firstObject && firstObject.image;
+  const title = firstObject && firstObject.title;
+  return (
+    <>
+      <div className="relative">
+        <img
+          src={
+            image
+              ? image
+              : "https://i.ytimg.com/vi/AO_aGmNEMME/maxresdefault.jpg"
```
Comment (anasm11) [inline src/components/cards/PlaylistCard.jsx:16]: You can use or (pipe) operator here.

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -57,38 +58,42 @@ export const Navbar = (props) => {
...
-                <FaHistory />
-              </Link>
-            </div>
+            {userLogin ? (
+              <>
+                <div
+                  title="Like"
+                  className="text-gray-300 hover:text-white text-2xl text-center px-2 py-2 m-2  cursor-pointer"
+                >
+                  <Link to="/liked">
```
Comment (bhtibrewal) [inline src/components/navbar/Navbar.jsx:67]: Suggestion: use Navlink here

**Review by anasm11 — APPROVED** [review]
Adding to playlist feature is working. You can also add the features to delete playlist videos and delete playlists.

**Review by bhtibrewal — APPROVED** [review]
The code and UI both looks good 👍 , you can go ahead and merge

**Pull request merged**